### PR TITLE
Change font size for portfolio item

### PR DESCRIPTION
**Pull request by abscondite**

### CODE REVIEW

**Comment by abscondite** [comment]
screen shot 2017-02-24 at 2 36 45 am

File: app/assets/stylesheets/_portfolio.scss [hunk 1]
```diff
@@ -111,10 +111,22 @@
...
           width: 20%;
         }
+
+        .item-title {
+          text-transform: uppercase;
+          color: grey;
+        }
+
+        .item-description {
+          font-weight: bold;
```
Comment (galulex) [inline app/assets/stylesheets/_portfolio.scss]: we do not have bold font connected. We use font-shadow instead.

**Comment by galulex** [comment]
Can we center the content vertically please?

**Comment by abscondite** [comment]
screen shot 2017-03-12 at 11 00 31 pm

**Pull request merged**